packages/cli/src/lib/inputHelpers.ts

```diff
@@ -15,38 +15,164 @@
  */
 
 import Color = require('color');
+import {URL} from 'url';
 import {isWebUri} from 'valid-url';
-import {util} from '@bubblewrap/core';
+import {Result, DisplayMode, asDisplayMode, util} from '@bubblewrap/core';
+import {ValidateFunction} from './Prompt';
+import {enUS as messages} from './strings';
+import {domainToASCII} from 'url';
 
-const MIN_KEY_PASSWORD_LENGTH = 6;
+/**
+ * A {@link ValidateFunction} that receives a {@link string} as input and resolves to a
+ * {@link Color} when successful.
+ * @param {string} color a string to be converted to a {@link Color}.
+ * @returns {Result<Color, Error} a results that resolves to a {@link Color} on success or
```

Comment:
nit: `Error` -> `Error>`

Reply:
(same a couple of times below as well)

Reply:
Yea, had this across the whole code. Fixed.